### ISSUE TITLE
Feature request: Imports on a new line

### ISSUE THREAD

**Issue by Ran4** [microsoft/pylance-release#1097]
If we have this code, where the cursor it at `|`:

```python3
from uuid import (
    UUID,
)

uuid4|
```

and choose to auto-import uuid4 from uuid, then I end up with

```python3
from uuid import (
    UUID, uuid4,
)

uuid4|
```

**Describe the solution you'd like**

In order to generate more readable git diffs, I much prefer this behaviour:

```python3
from uuid import (
    UUID,
    uuid4,
)

uuid4|
```

It would be really cool if there was a way of customizing this behaviour.

**Comment by erictraut** [comment]
I've added a new heuristic so the auto-import text honors the existing formatting.

```python
from uuid import UUID
```
becomes
```python
from uuid import UUID, uuid4
```

and
```python
from uuid import (
    UUID,
)
```
becomes
```python
from uuid import (
    UUID,
    uuid4,
)
```

This avoids the need for an explicit setting.

This change will be in the next release of pylance.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.3.4, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202134-31-march-2021
